io/io/test/TFileMergeClientServer.cxx

```diff
@@ -0,0 +1,84 @@
+#include <memory>
+#include <random>
+#include <thread>
+
+#include "gtest/gtest.h"
+
+#include "TFile.h"
+#include "TROOT.h"
+#include "TTree.h"
+
+#include "ROOT/TFileMergeClient.hxx"
+#include "ROOT/TFileMergeServer.hxx"
```

Comment:
Sure, I will reorder them. I had them like that before, in fact, and only `TFileMergeServer.hxx` is needed here also.